static/js/ui.js

```diff
@@ -203,9 +203,13 @@ $(function () {
         $("#normal-zephyr-mirror-error-text").addClass("notdisplayed");
         $("#desktop-zephyr-mirror-error-text").removeClass("notdisplayed");
     }
-});
+};
 
 exports.initialize = function () {
+    exports.set_compose_textarea_handlers();
+
```

Comment:
I find this new line spacing weird. But, maybe that's just me.

Reply:
I sorta agree, though putting them together also felt weird, so I decided to not worry about it and merge.